scripts/restart_mcp_server.sh

```diff
@@ -6,19 +6,67 @@ CONTAINER_NAME=${CONTAINER_NAME:-stt-mcp-server-linux}
 TMUX_SESSION=${TMUX_SESSION:-claude}
 TMUX_TMPDIR=${TMUX_TMPDIR:-"$HOME"/.tmux}
 
-docker stop "$CONTAINER_NAME" || true
-docker rm "$CONTAINER_NAME" || true
+# Check if Docker is running
+if ! docker info >/dev/null 2>&1; then
+    echo "Error: Docker is not running or not accessible" >&2
+    exit 1
+fi
+
+# Stop and remove container if it exists
+if docker ps -a --format '{{.Names}}' | grep -q "^${CONTAINER_NAME}$"; then
+    echo "Stopping and removing existing container: $CONTAINER_NAME"
+    docker stop "$CONTAINER_NAME" || true
+    docker rm "$CONTAINER_NAME" || true
+else
+    echo "Container $CONTAINER_NAME does not exist, proceeding with creation"
+fi
+
+# Create required directories
```

Comment:
Please remove the directory creation, it's already performed by https://github.com/marcindulak/stt-mcp-server-linux/blob/922d09a65e5f650aef4bce6be33b4794dcc5b72f/scripts/add_mcp_server_to_claude.sh#L8-L10